web/apps/labelstudio/src/pages/CreateProject/Config/Config.jsx

```diff
@@ -40,36 +41,28 @@ const Label = ({ label, template, color }) => {
 
   return (
     <li className={configClass.elem("label").mod({ choice: label.tagName === "Choice" })}>
-      <label style={{ background: color }}>
-        <Input
-          type="color"
-          className={configClass.elem("label-color")}
-          value={colorNames[color] || color}
-          onChange={(e) => template.changeLabel(label, { background: e.target.value })}
-        />
-      </label>
-      <span>{value}</span>
-      <button
+      <span className="flex">
+        <label style={{ background: color }}>
+          <input
```

Comment:
we're reveting back to native input? is that intended?

Reply:
reminder about this. i would say this is not the recommended way to go - we need to reuse our components